api/spec/requests/spree/api/orders_controller_spec.rb

```diff
@@ -815,6 +815,38 @@ module Spree
           expect(response.status).to eq 201
           expect(json_response["user_id"]).to eq(user.id)
         end
+
+        context 'with payment', js: true do
+          let(:params) do
+            {
+              payments: [{
+                amount: '10.0',
+                payment_method: payment_method.name,
+                source: {
+                  month: "01",
+                  year: Date.today.year.to_s.last(2),
+                  cc_type: "123",
+                  last_digits: "1111",
+                  name: "Credit Card"
+                }
+              }]
+            }
+          end
+          let!(:payment_method) { create(:payment_method) }
```

Comment:
I think that, since you're using `payment_method` only once, you can inline this `let` directly where it's used and save one line without compromising readability: 

```ruby
amount: '10.0',
payment_method: create(:payment_method).name,
source: {
```

Reply:
done here https://github.com/solidusio/solidus/pull/3066/files#diff-72468e52a59aedfe8a3443597f1a6f3aR836